apm-agent-plugins/apm-servlet-plugin/src/main/java/co/elastic/apm/agent/servlet/AsyncInstrumentation.java

```diff
@@ -108,7 +108,10 @@
             private static final AsyncContextAdviceHelper<AsyncContext> asyncHelper = new AsyncContextAdviceHelperImpl(GlobalTracer.requireTracerImpl());
 
             @Advice.OnMethodExit(suppress = Throwable.class, inline = false)
-            public static void onExitStartAsync(@Advice.Return AsyncContext asyncContext) {
```

Comment:
fixed potential NPE, for example when trying to start an asynchronous servlet that does not support async (very very unlikely in production).